### DISCUSSION TITLE
Collapse comment thread

### DISCUSSION THREAD

**Discussion by thumDer** [outline/outline#7330] [Ideas]
I think it would be really handy, if comment threads could be collapsable, without resolving them. Some of our thread got quite long (with long comments), but far from completion, and it can get really difficult to scroll through them for an overview.
Extra option would be a collapse / expand all at the top next to show resolved.

**Comment by tommoor** [comment]
Yea, this was part of the original spec but got cut for scope – still a tracking issue here:
https://github.com/outline/outline/issues/5079

**Reply by thumDer** [reply]
great, hope it gets implemented!
